ferrocene/doc/release-notes/src/25.02.0.rst

```diff
@@ -8,12 +8,19 @@ Ferrocene 25.02.0
 
 Ferrocene 25.02.0 is the fifth major release of Ferrocene.
 
-The highlights of this release are the inclusion of Rust 1.83.0, as well as several new experimental targets.
+The highlights of this release are the inclusion of Rust 1.83.0, as well as several new 
+experimental targets.
 
 New features
 ------------
 
-Updated the Rust version to include changes in Rust 1.82.0 and 1.83.0.
+* Updated the Rust version to include changes in Rust 1.82.0 and 1.83.0.
+* Quality Managed support has been added for a new target. Quality Managed targets are not 
+  qualified for safety critical use, but are otherwise fully tested and supported. The new
+  target is:
```

Comment:
Should we mention it is tier 1 to be consistent with rustc terminology?

Reply:
No. We do not equate Ferrocene support tiers to Rust tiers.